style.css

```diff
@@ -0,0 +1,73 @@
+body {
+	background-color: #ffffe6;
+}
+h1 {
+	font-family: 'Kavoon', cursive;
+	/*font-family: 'Oleo Script Swash Caps', cursive;*/
+	/*font-family: 'Galada', cursive;*/
+	/*font-family: 'Monofett', cursive;*/
+	/*font-family: 'Akronim', cursive;*/
+	font-size: 2.5em;
+	text-align: center;
+	color: #00004d;
+}
+#loading {
+	text-align: center;
+	font-size: 1.5em;
+	visibility: hidden;
+}
+#buttons {
+	margin: auto;
+	text-align: center;
+}
+.btnArea {
+	color: #00004d;
+	display: inline-block;
+	/*width: 100%;*/
+	text-align: center;
+	font-size: 1.2em;
+	border-radius: 10px;
+	background-color: #cce6ff;
+	padding: 0.4em;
+	box-shadow: 0 0px 0px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+.btnArea:hover {
+	box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24), 0 17px 50px 0 rgba(0,0,0,0.19);
+	cursor: pointer;
+	transition-duration: 0.8s;
+	border-color: #00004d; 
+}
+.info {
+	margin: 25px auto;
+	border: 2px solid #005cb3;
+	background-color: #cce6ff;
+	border-radius: 25px;
+	width: 60%;
+	padding: 25px;
+	font-size: 1.8em;
+	color: #000033;
+	word-wrap: break-word;
+	/*text-overflow: string;*/
+	/*white-space: normal;*/
+	/*overflow: hidden;*/
+}
+@media screen and (max-width: 321px) {
+	body {
+		font-size: 75%;
+	}
+	.info {
+		width: auto;
+	}
+	.btnArea {
+	display: block;
```

Comment:
Not sure which editor you use? but the white spacing consistency is a little off